### PR TITLE
docs: Added bug report and feature requests templates

### LINKED ISSUE #26: Add issue templates
I noticed there are no issue templates for better Github workflow. Can I add the templates?

### PR DESCRIPTION
**Pull request by Abjcodes**
Added bug report and feature requests templates in the .github/ISSUE_TEMPLATE folder.
Fixes #26

### CODE REVIEW

**Comment by s-ayush2903** [comment]
Please update the PR description as `Fixes #issue_num`, doing so will close the related issue as soon as this is merged.

File: .github/ISSUE_TEMPLATE/BUG_REPORT_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve the project
+
+---
+</br>
+ 
+### Steps to reproduce 
+1. Write here
+2. Write here
```
Comment (s-ayush2903) [inline .github/ISSUE_TEMPLATE/BUG_REPORT_TEMPLATE.md:11]: Change stuff like this to comments

File: .github/ISSUE_TEMPLATE/FEATURE_REQUESTS_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ 
+### Is your feature request related to a problem? Please describe.
+(Please present a concise description of the problem to be addressed by this feature request.) 
+</br></br>
+
+### Describe the solution you'd like
+(A clear and concise description of what you want to happen.)
+* Details of the technical implementation
+* Tradeoffs made in design decisions
+* Caveats and considerations for the future
```
Comment (s-ayush2903) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUESTS_TEMPLATE.md:19]: Same, move this inside the comments block, as people will have to delete it all the time they create an issue, this should be as a hint/info for 'em on how to dress an issue desc.

File: .github/ISSUE_TEMPLATE/FEATURE_REQUESTS_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+</br></br>
+
+### Additional context
+(Add any other context or screenshots about the feature request.)
```
Comment (s-ayush2903) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUESTS_TEMPLATE.md:25]: Same

**Comment by Abjcodes** [comment]
Sure I will make the necessary changes.

**Review by s-ayush2903 — APPROVED** [review]

**Pull request merged**